_techwatch/network.md

```diff
@@ -12,3 +12,9 @@ layout: page
 * auto-gen TOC:
 {:toc}
 
+## Working Documents ##
+*   https://twiki.cern.ch/twiki/bin/view/HEPIX/TechwatchNetwork/WebHome
+   
+## Meetings ##
+*   On Indico: https://indico.cern.ch/category/10621/
```

Comment:
Same as above.

If you are not familiar with GitHub, you can fix the text, commit it and just push it to your branch.